L1C_Aux/hirs_footprint.py

```diff
@@ -46,8 +46,21 @@ def getgacpix(hirs_spot, dtime):
     #----------------------------
     h = 819.      #Nominal METOP-A orbital height
     R = 6372.     #Earth Radius
-    speed = 6.7   #Ground-track speed in km.s-1    
-    IFOV = math.radians(1.22) # HIRS/2 IFOV ~= 0.0213 radians.
+    speed = 6.7   #Ground-track speed in km.s-1
+
+    HIRS2 = ('noaa08','noaa09','noaa10','noaa12','noaa14')
+    HIRS2i = ('noaa11','noaa14')
+    HIRS3 = ('noaa15','noaa16','noaa17')
+    HIRS4 = ('noaa18','noaa19','metopa','metopb')
+
+
+    if hirs_sensor in HIRS2:   
```

Comment:
Could you make these lines of the form: if hirs_sensor.lower() in []?  This is used also in read_functions.py and provides an extra safety net if the command line input is in upper case.

Reply:
Resolved with new commit.